db/models/postModel.js

```diff
@@ -0,0 +1,16 @@
+import { Sequelize } from 'sequelize';
+
+
+let Post = Sequelize.define('user', {
+  title: {
+  	type: Sequelize.STRING,
+  	unique: true
+
+  },
+  content: {
+  	type:Sequelize.TEXT,
+  	allowNull: false,
```

Comment:
Think you might have an extra comma inside the object literal, otherwise, this request is ready to be merged!  Nice work Maurice.